yarn.lock

```diff
@@ -9863,6 +9863,40 @@ amdefine@>=0.0.4:
   resolved "https://registry.npmjs.org/amdefine/-/amdefine-1.0.1.tgz#4a5282ac164729e93619bcfd3ad151f817ce91f5"
   integrity sha1-SlKCrBZHKek2Gbz9OtFR+BfOkfU=
 
+amplify-cli-core@4.0.1:
+  version "4.0.1"
+  resolved "https://registry.npmjs.org/amplify-cli-core/-/amplify-cli-core-4.0.1.tgz#86167d3c54264448110c95ce872d6c338b911975"
```

Comment:
can you run `yarn why amplify-cli-core` and check where this is coming from.
We shouldn't have this entry.

Reply:
Running this did not show any dependencies